src/styles/Task.scss

```diff
@@ -2,10 +2,11 @@
   border-radius: 10px;
   display: flex;
   flex-direction: column;
-  margin: 1vh auto;
-  margin-bottom: 5vh;
+  margin: 0 132px;
```

Comment:
🤔

Reply:
Lol, yeah my CSS is a bit sus. Uh, maybe I'll add a TODO to the backlog of changing this `132px` margin for responsive design.